lib/email_api/email_template_schemas.yaml

```diff
@@ -32,3 +32,11 @@ shutoff_error:
     errored_table: "No Errored VMs"
   html_filepath: "html/shutoff_error.html.j2"
   plaintext_filepath: "plaintext/shutoff_error.txt.j2"
+
+decom_email:
+  schema:
+    username: null
+    affected_flavors: "No Flavors"
+    decom_table: "No VMs affected"
```

Comment:
Surely these should be null, rather than email people saying "nothing to do"?

Reply:
either way this won't be seen unless we're running some obscure tests